src/lib/rollup.ts

```diff
@@ -77,6 +78,7 @@ const build = async (
 				},
 			}),
 			typescript({ sourceMap: false }),
+			commonjs(),
```

Comment:
I am a bit unclear on why we need `commonjs` from rollup. I know rollup is a bundler but what is considered CommonJS modules in here?

Reply:
The source package imports some commonjs so without adding this to rollup the import doesn't get correctly compiled and none of the templates will build. Feel free to play around with it locally but we do need it!

Reply:
I now understand after you explained that source package imports some commonjs and that's why we obviously need it. I will do that now and let you know if I have more questions.